### ISSUE TITLE
API Change: qx serve <target> instead of qx serve <configFile>

### ISSUE THREAD

**Issue by cboulanger** [qooxdoo/qooxdoo-compiler#555]
I think that very few people actually uses `qx serve <configFile>`, so we could easily move this use to a parameter such as `qx serve --use-config <configFile>`. A much more useful syntax would be `qx serve <target>`, to save typing `qx serve --target=<target>`.

What do you think?

**Comment by cboulanger** [comment]
The change could involve some shim which would silently move a file parameter to the correct option.

**Comment by johnspackman** [comment]
fine by me

**Comment by hkollmann** [comment]
What‘s about qx compile configFile? It‘s the same syntax. When we change the synrax we should change it consistent. Changing in qx compile break build chains.

Von meinem iPad gesendet

**Comment by cboulanger** [comment]
yes, same thing for `qx compile`. In order not to break things, the new syntax would silently allow the old by testing if the first non-parameterized argument is the path to a file and move it to the parameter.

**Comment by hkollmann** [comment]
IMHO we can break things in this state of the project if we anounce it. The change in the build steps are minimal.

Von meinem iPad gesendet
